sui/token_bridge/sources/create_wrapped.move

```diff
@@ -1,23 +1,26 @@
 /// This module uses the one-time witness (OTW)
 /// Sui one-time witness pattern reference: https://examples.sui.io/basics/one-time-witness.html
 module token_bridge::create_wrapped {
-    use sui::coin::{Self};
     use std::string::{Self};
+    use std::ascii::{Self};
     use std::option::{Self};
+    use sui::coin::{Self, CoinMetadata};
     use sui::transfer::{Self};
     use sui::tx_context::{TxContext};
     use sui::url::{Url};
     use wormhole::state::{Self as wormhole_state, State as WormholeState};
     use wormhole::myvaa as core_vaa;
 
-    use token_bridge::asset_meta::{Self};
+    use token_bridge::asset_meta::{Self, symbol_to_string, name_to_string};
```

Comment:
I think we should just import the module and use `asset_meta::symbol_to_string` and `asset_meta::name_to_string`. Little easier to read the code with the module qualifiers IMO (and that's how these two functions are already used in this module)